### ISSUE TITLE
Deprecated dependencies

### ISSUE THREAD

**Issue by nwwells** [wistityhq/strapi#112]
Just running through the start up guide on node v6. Got the following warnings:

```
npm WARN deprecated koa-gzip@0.1.0: please use koa-compress instead.
npm WARN deprecated native-or-bluebird@1.2.0: 'native-or-bluebird' is deprecated. Please use 'any-promise' instead.
npm WARN deprecated lodash@2.4.1: lodash@<3.0.0 is no longer maintained. Upgrade to lodash@^4.0.0.
npm WARN deprecated graceful-fs@3.0.8: graceful-fs v3.0.0 and before will fail on node releases >= v6.0. Please update to graceful-fs@^4.0.0 as soon as possible. Use 'npm ls graceful-fs' to find it in the tree.
```

Here's the relevant parts of the dependency tree:

```

  │ ├─┬ koa-send@1.3.1
  │ │ └─┬ mz@1.3.0
  │ │   └── native-or-bluebird@1.2.0
  ├─┬ sails-disk@0.10.9
  │ │...
  │ ├── lodash@2.4.1
  │ ├─┬ waterline-criteria@1.0.0
  │ │ └── lodash@2.4.2
  │ ├─┬ waterline-cursor@0.0.6
  │ │ ├── async@0.9.2
  │ │ └── lodash@2.4.2
  ├─┬ strapi-generate@1.5.0
  │ └─┬ reportback@0.1.9
  │   ├─┬ captains-log@0.11.11
  │   │ ├── lodash@2.4.1
  │   │  ...
  │   ├── lodash@2.4.2
  │   ├─┬ merge-defaults@0.1.4
  │   │ └── lodash@2.4.2
  │   └─┬ switchback@1.1.3
  │     └── lodash@2.4.2
  ├─┬ unzip2@0.2.5
  │ │ ...
  │ ├─┬ fstream@0.1.31
  │ │ └── graceful-fs@3.0.8
  ├─┬ waterline@0.10.31
        ...
  │ ├─┬ switchback@2.0.0
  │ │ └── lodash@2.4.2
  │ ├─┬ waterline-criteria@0.11.2
  │ │ └── lodash@2.4.2

**Comment by ajayphp** [comment]
when v 2.0 expected,
or atleast fix this npm WARN deprecated koa-gzip@0.1.0: please use koa-compress instead.

I am new to MEAN stack and found you to start on app, but following errors,
as this is based on koa and you did not update , is a warning not to use on production as
may break ...

i used with bitnami mean stack, that has node js 6.0
is this stack will work with node 6.0 or which older version to work onthis


npm WARN deprecated native-or-bluebird@1.2.0: 'native-or-bluebird' is deprecated. Please use 'any-promise' instead.
npm WARN deprecated lodash@2.4.1: lodash@<3.0.0 is no longer maintained. Upgrade to lodash@^4.0.0.
npm WARN deprecated graceful-fs@3.0.8: graceful-fs v3.0.0 and before will fail on node releases >= v6.0. Please update to graceful-fs@^4.0.0 as soon as possible. Use 'npm ls graceful-fs' to find it in the tree.

**Comment by Aurelsicoko** [comment]
It should be fixed with the v3.0@alpha. As we said in our latest blog post, the v2.0 has been released but only for people which need it. Please don't start a new project on the v2. Use the v1.5.4 at this time.
